### PR TITLE
adding glob to label-has-associated-control

### PR DESCRIPTION
**Pull request by lilia-paz**
fixes issue #720 

Request to add glob or regex to controlComponents in label-has-associated-control. Added glob to components

### CODE REVIEW

File: src/rules/label-has-associated-control.js [hunk 1]
```diff
@@ -58,12 +58,12 @@ module.exports = {
...
-        'output',
-        'progress',
-        'select',
-        'textarea',
+        '*Input',
+        '*Meter',
+        '*Output',
+        '*Progress',
+        '*Select',
+        '*Textarea',
```
Comment (ljharb) [inline src/rules/label-has-associated-control.js:66]: Thanks for the PR! I'm not sure how this actually resolves the issue tho - these are just the defaults (which should not have asterisks). `mayContainChildComponent` is what likely needs to be tweaked to support stars.

Comment (lilia-paz) [reply src/rules/label-has-associated-control.js:66]: Hi Jordan, thank you for your comments. I'm looking at modifying <mayContainChildComponent> and including the glob package, picoMatch. Does this sound good?

Comment (lilia-paz) [reply src/rules/label-has-associated-control.js:66]: Hi Jordan, thanks for your comment. I'm going to modify `mayContainChildComponent` and was going to include the package, picoMatch, for globs. Does this work? Thanks again.

Comment (ljharb) [reply src/rules/label-has-associated-control.js:66]: I'd prefer using minimatch, but sounds great thanks

**Comment by jessebeach** [comment]
Thank you for the PR @lilia-paz . Please reopen this PR when you plan to continue working on it.

**Pull request closed without merging**